research-hub-web/src/app/app.component.scss

```diff
@@ -38,6 +38,27 @@
   right: 8px;
 }
 
+/*
+ * Banner Styling
+ */
+ .banner-container-home {
+  text-align: center;
+  min-width: 100vw; 
+  margin: -3em 0PX 20PX -7.5vw;
```

Comment:
Alright, bit of a ramble here: Not a fan of negative margins just to get the full bleed effect. If you really had to do something like this then I would definitely be using `calc()` with the `$site-padding` variable.

I think that's still a pretty bad idea though. Here's what I propose, as full-bleed containers will be a pretty common use case here's my suggestion:

In the root `styles.scss` file add the following new class:

```scss
.full-bleed {
  width: 100vw;
  position: relative;
  left: 50%;
  right: 50%;
  margin-left: -50vw;
  margin-right: -50vw;
}
```

There's quite a few different opinions on the best way to create full-bleeds, [you can read here](https://css-tricks.com/full-bleed/) if you're interested but that's what I usually use in personal projects. Then you can just use it anywhere you want a full-bleed element, in this case:

```html
  <!-- Banner -->
                <div class="banner-container-home full-bleed" *ngIf="showBanner">
```

would do the trick.